src/search/components/SearchResults.tsx

```diff
@@ -2,8 +2,8 @@ import React, { FunctionComponent } from 'react';
 
 import { Blankslate, Container, Flex, Pagination, Spacer, Spinner } from '@viaa/avo2-components';
 
-import SearchResultItem from './SearchResultItem';
-import { SearchResultsProps } from './types';
+import { SearchResultsProps } from '../search.types';
+import { SearchResultItem } from './';
```

Comment:
Best niet importen vanuit de index in dezelfde folder om circular dependencies te vermijden